subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -1222,13 +1241,17 @@ static void set_bn_max_pdu(bool framed, uint32_t iso_interval,
 		}
 	} else {
 		/* For unframed, ISO_Interval must be N x SDU_Interval */
-		LL_ASSERT(iso_interval % sdu_interval == 0);
+		if ((iso_interval % sdu_interval) != 0) {
+			iso_interval += iso_interval_cig;
+		}
```

Comment:
Can you help me understand this? 

If the ISO interval is not a multiple of SDU interval, then we add the `iso_interval_cig`? 

The comment is not really making sense anymore, but I also don't understand this addition of `iso_interval_cig`